.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+docker.image('debian').withRun('-v /var/run/docker.sock:/var/run/docker.sock'){
+
+    checkout scm
+    sh 'ls'
+    sh "sudo apt-get update && sudo apt-get install -y docker-ce"
+    sh "rm go1* && wget https://storage.googleapis.com/golang/go1.15.6.linux-amd64.tar.gz"
```

Comment:
You shouldn't need to "rm go1*" because this is a newly started container and will have nothing in this directory.

Reply:
Hello Robert, thanks for leaving these comments. I have refactored the Jenkinsfile so that it will look cleaner. I have also added a clean up method so I won't need the **rm** here anymore.